lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -375,6 +389,60 @@ function createPsuedoLocaleStrings(messages) {
   return psuedoLocalizedStrings;
 }
 
+/**
+ * @param {import('typescript').NamedDeclaration} node
+ * @return {string}
+ */
+function getIdentifier(node) {
```

Comment:
Explanation of what this does?  Seems like a tsc helper method to get the text for an ast property?

Reply:
> Explanation of what this does? Seems like a tsc helper method to get the text for an ast property?

done